components/automate-cli/cmd/chef-automate/reindex-start.go

```diff
@@ -0,0 +1,106 @@
+package main
```

Comment:
Any specific reason for naming this file as reindex-start.go? If not please rename it to reindex.go